lte/gateway/c/core/oai/tasks/nas/ies/ApnAggregateMaximumBitRate.c

```diff
@@ -86,22 +86,22 @@ int encode_apn_aggregate_maximum_bit_rate(
   encoded++;
 
   if (apnaggregatemaximumbitrate->extensions &
-      APN_AGGREGATE_MAXIMUM_BIT_RATE_MAXIMUM_EXTENSION_PRESENT) {
+          APN_AGGREGATE_MAXIMUM_BIT_RATE_MAXIMUM_EXTENSION_PRESENT |
```

Comment:
can you add unit test? see the example in directory https://github.com/magma/magma/tree/master/lte/gateway/c/core/oai/test/mme_app_task